lib/OptimizationStages.js

```diff
@@ -0,0 +1,10 @@
+/*
+	MIT License http://www.opensource.org/licenses/mit-license.php
+	Author Florent Cailhol @ooflorent
+*/
+
+"use strict";
+
+exports.STAGE_BASIC = 0;
+exports.STAGE_DEFAULT = 10;
```

Comment:
For backward-compat it would be great if the default is 0 (basic = -10).

Reply:
Yep. I was hesitant. That's probably the best move.